App/screens/HomeScreen.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { View, Text, StyleSheet, ScrollView, TouchableOpacity, Image, ImageBackground } from 'react-native';
+import Svg, { Rect, LinearGradient, Stop, Defs } from 'react-native-svg';
+import ProgressCard from '../components/ProgressCard';
+import CompletedPathCard from '../components/CompletedPathCard';
+import { typography } from '../styles/typography';
+
+
+const HomeScreen = () => {
+
+ 
+
+
+    const inProgressPaths = [
+        { number: 14, ang: 745, progress: 65 },
+        { number: 15, ang: 1145, progress: 25 },
+        { number: 16, ang: 500, progress: 40 },
+        { number: 17, ang: 800, progress: 80 },
+    ];
+
+    const completedPaths = [
+        { number: 11, date: "4th Aug 2024" },
+        { number: 10, date: "13th Jan 2024" },
+        { number: 9, date: "11th Oct 2023" },
+    ];
+
+    return (
+
+        <ImageBackground
+            source={require('../assets/background.png')}
+            style={styles.backgroundImage}
+        >
+            <ScrollView style={styles.container}>
```

Comment:
Use view here
also wrap section into view like wrap content from line no 34 to 54 and then from 55 to 70 and then 71 to 85